x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_grouped_findings_evaluation.ts

```diff
@@ -27,10 +27,10 @@ export interface ResourceTypeBucket extends KeyDocCount {
   };
 }
 
-export const resourceTypeAggQuery = {
+export const failedFindingsAggQuery = {
```

Comment:
this should be named `cisSectionAggsQuery`. in general, I would recommend trying and use more specific namings in this PRs. it's a complicated line to draw so I'm approving anyway but it would be really nice if atleast some of the functions and files will be more specific towards why they try to accomplish